packages/gatsby-plugin-image/README.md

```diff
@@ -136,7 +91,7 @@ const width = 300
 
 ## Installation
```

Comment:
If we change the installation instructions in the usage section we don't need this anymore.